src/dbSta/include/db_sta/dbReadVerilog.hh

```diff
@@ -56,6 +59,24 @@ using odb::dbDatabase;
 
 using sta::NetworkReader;
 
+
```

Comment:
I seriously doubt there is any reason to export this class.
It implements the sta::Network api, so you should just include and use that.